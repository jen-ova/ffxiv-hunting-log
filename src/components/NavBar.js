import React, {useState} from "react";
import Wrapper from "./Wrapper";
import "../styles/navBar.css";

const NavBar = () => {

  return (
    <Wrapper>
      <div className="navBar-container">
        <h1 className="navBar-container__title">FFXIV Hunting Log</h1>
        <div className="navBar-container__account">
          {/* <i className="fas fa-filter"></i> */}
          {/* <i className="fas fa-bullseye"></i> */}
          {/* <i className="fas fa-user"></i> */}
        </div>
      </div>
    </Wrapper>
  );
};

export default NavBar;
