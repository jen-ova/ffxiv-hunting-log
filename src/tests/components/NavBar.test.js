import React from "react";
import { render } from "@testing-library/react";
import NavBar from "../../components/NavBar";

describe("NavBar", () => {
  it("renders correctly", () => {
    const { asFragment } = render(<NavBar />);
    expect(asFragment()).toMatchSnapshot();
  });
});
