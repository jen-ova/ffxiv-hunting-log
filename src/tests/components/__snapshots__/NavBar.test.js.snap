// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavBar renders correctly 1`] = `
<DocumentFragment>
  <div>
    <div
      class="navBar"
    >
      <p>
        FFXIV Hunting Log
      </p>
      Login / Logout / Register
    </div>
  </div>
</DocumentFragment>
`;
